src/components/ListLayout.jsx

```diff
@@ -61,6 +62,12 @@ const ListLayout = ({ items, localToken }) => {
     return timeCheck;
   }
 
+  function handleClick(itemId, itemName) {
+    if (window.confirm(`Are you sure you want to delete ${itemName}?`)) {
```

Comment:
Great on using system built in delete dialog instead of custom one - that way OS will handle accessibility and default behaviors for you.